src/textual/widgets/_select.py

```diff
@@ -278,6 +278,44 @@ def __init__(
         self._value: SelectType | None = value
         self._options = options
 
+    @classmethod
+    def from_values(
+        cls,
+        opts: [SelectType],
```

Comment:
I don't think I've ever seen this syntax `opts: [SelectType]` to declare a list of `SelectType` objects, but we don't want to restrict ourselves to lists as inputs!

Please consider changing the typing to accept a sequence of `SelectType`, instead of only a list.

Reply:
I changed to Iterator[SelectType], which I think is more general than a list?  Let me know if this is correct or if it should be something else.

Reply:
`Iterable[SelectType]` was the correct choice, I think. Thanks.